src/Components/ExpandedRulesDetails.js/ExpandedRulesDetails.js

```diff
@@ -17,18 +17,24 @@ import {
 import { Table, Tbody, Td, Th, Thead, Tr } from '@patternfly/react-table';
 import PropTypes from 'prop-types';
 import TemplateProcessor from '@redhat-cloud-services/frontend-components-advisor-components/TemplateProcessor/TemplateProcessor';
+import ObjectsModal from '../ObjectsModal/ObjectsModal';
+import { ObjectsTableColumns } from '../../AppConstants';
 
-const columnNames = {
-  object: 'Object ID',
-  kind: 'Kind',
-};
 const code = `oc get namespace -o jsonpath={range .items[*]}{.metadata.name}{"\t"}{.metadata.uid}{"\n"}{end}
   oc -n <namespace> get <resourceKind> -o jsonpath={range .items[*]}{.metadata.name}{"\t"}{.metadata.uid}{"\n"}{end}`;
 
 const ExpandedRulesDetails = ({ more_info, resolution, objects }) => {
+  const [objectsModalOpen, setObjectsModalOpen] = useState(false);
   return (
     <Card className="ins-c-report-details" style={{ boxShadow: 'none' }}>
       <CardBody>
+        {objectsModalOpen && (
```

Comment:
We are doing the condition already in the modal
```suggestion
```

Reply:
Thanks, removed it